src/de/jost_net/JVerein/gui/parts/LesefeldUebersichtPart.java

```diff
@@ -74,83 +76,21 @@ public void paint(final Composite parent) throws RemoteException
 
     container.addHeadline("Lesefelder");
+    container.addLabelPair("Mitglied", getMitglied());
 
-    // Hole alle Mitglieder aus Datenbank um sie sp�ter anzuzeigen.
-    DBIterator<Mitglied> it = Einstellungen.getDBService()
-        .createList(Mitglied.class);
-    // TODO Wenn es "zu viele" Mitglieder gibt, ist ein SelectInput
-    // nicht geeignet. Es sollte eine andere Art der Auswahl eingebaut
-    // werden.
-    it.setOrder("order by name, vorname");
-    // optional k�nnten Filter eingebaut werden:
-    // it.addFilter("plz='" + (String) plz.getValue() + "'");
-    ArrayList<Mitglied> mitgliederList = new ArrayList<>();
-    while (it.hasNext())
+    if (selectedMitglied == null && getMitglied().getValue() != null)
     {
-      mitgliederList.add(it.next());
+      selectedMitglied = (Mitglied) getMitglied().getValue();
     }
-
-    // Das erste Mitglied wird ausgew�hlt, wenn im Contructor keins gew�hlt
-    // wurde.
-    if (selectedMitglied == null && !mitgliederList.isEmpty())
-    {
-      selectedMitglied = mitgliederList.get(0);
-    }
-    if (selectedMitglied == null)
-    {
-      Label textLabel = new Label(container.getComposite(), SWT.NONE);
-      textLabel.setText("Bitte erst ein Mitglied anlegen.");
-      return;
-    }
-
-    // MITGLIEDER AUSWAHL-FELD
-    final SelectInput mitgliederSelectInput = new SelectInput(
-        mitgliederList.toArray(), selectedMitglied);
-
-    mitgliederSelectInput.addListener(new Listener()
-    {
-
-      @Override
-      public void handleEvent(Event event)
-      {
-        Mitglied selected = (Mitglied) mitgliederSelectInput.getValue();
-        if (selected == null || selected == selectedMitglied)
-          return;
-
-        selectedMitglied = selected;
-        try
-        {
-          lesefeldAuswerter
-              .setMap(new MitgliedMap().getMap(selectedMitglied, null, true));
-          lesefeldAuswerter.evalAlleLesefelder();
-          List<Lesefeld> lesefelder = lesefeldAuswerter.getLesefelder();
-          for (Lesefeld lesefeld : lesefelder)
-          {
-            updateLesefeldEinstellungRow(lesefeld);
-          }
-        }
-        catch (RemoteException e)
-        {
-          String fehler = "Fehler beim Ausw�hlen des Mitgliedes";
-          Logger.error(fehler, e);
-          GUI.getStatusBar().setErrorText(fehler);
-        }
-
-      }
-    });
-    mitgliederSelectInput.setName("Mitgliedschaft");
-    container.addLabelPair("Mitglied", mitgliederSelectInput);
-    // ENDE MITGLIEDER AUSWAHL-FELD
-
     // LesefelderListeLayout
     // darf nur �ber die Funktionen
     // addLesefeldEinstellungRow(), updateLesefeldEinstellungRow()
     // und deleteLesefeldEinstellungRow()
     // manipuliert werden.
     lesefelderListeLayout = new ColumnLayout(container.getComposite(), 4);
-
     lesefeldAuswerter = new LesefeldAuswerter();
     lesefeldAuswerter.setLesefelderDefinitionsFromDatabase();
+    if (selectedMitglied != null)
```

Comment:
Ich würde hier das erste Mitglied per Default auswählen da sonst das Lesefeld ohne Mitglied ausgeführt wird.
Eigentlich gehört der Select auch in die DetailView, so könnte man die Ausgab bei verschiedenen Mitgliedern testen.
Insgesamt sollten wir irgendwann die Lesefelder Views überarbeiten, damit sie mehr in unser gewohntes raster passt.

Reply:
Das erste Mitglied wird ausgewählt wenn man die Liste Option hat.
Bei der Suche Option gibt es kein Query nach Mitglieder, da habe ich keines.
Aber ja, ich könnte im Mitglied Input das Query nach Mitgliedern vor dem switch machen und das erste Mitglied bestimmen. Das könnte ich dann im Suche Fall benutzen.
Das kann ich machen.

Reply:
> Eigentlich gehört der Select auch in die DetailView, so könnte man die Ausgab bei verschiedenen Mitgliedern testen.

Mache ich machen

Reply:
Ist gemacht